storage/schema.go

```diff
@@ -122,10 +126,25 @@ type Job struct {
 	Tasks   []*Task  `json:"Tasks" gorm:"ForeignKey:JobID"`
 }
 
+type Worker struct {
+	ID      string  `json:"Id"`
+	VCPU    float32 `json:"Vcpu"`
```

Comment:
why the amount of vcpu is float?

Reply:
virtualization allows us to use float point numbers in the resources amount, so, 1.5 is an available amount for vcpu, for example.